dp2/config.yml

```diff
@@ -1,18 +1,20 @@
-#WS CONFIGURATION
+# Pipeline's webservice host
 host: http://localhost
+# Pipeline's webservice port
 port: 8181
```

Comment:
A note saying that these should be (left commented out if one wishes to use `app_path`, would be useful.